jetbrains_projects/__init__.py

```diff
@@ -84,15 +84,22 @@ def _parse_recent_projects(self, recent_projects_file: Path) -> list[Project]:
 
             projects = []
             for entry in entries:
-                project_path = entry.attrib["key"].replace("$USER_HOME$", str(Path.home()))
-
+                project_path = entry.attrib["key"]
+                files = Path(project_path + "/.idea").glob("*.iml")
+                project_path = project_path.replace("$USER_HOME$", str(Path.home()))
+                project_name = Path(project_path).name
                 tag_opened = entry.find(".//option[@name='projectOpenTimestamp']")
                 last_opened = tag_opened.attrib["value"] if tag_opened is not None and "value" in tag_opened.attrib else None
 
                 if project_path and last_opened:
                     projects.append(
                         Project(name=Path(project_path).name, path=project_path, last_opened=int(last_opened))
```

Comment:
refactor: you can also use `project_name` here instead of  `Path(project_path).name`